osu.Game.Rulesets.Osu/Edit/OsuHitObjectComposer.cs

```diff
@@ -128,6 +135,9 @@ protected override void Update()
             }
         }
 
+        public override float GetBeatSnapDistanceAt(HitObject referenceObject)
+            => (float)(base.GetBeatSnapDistanceAt(referenceObject) * DistanceSpacingMultiplier.Value);
```

Comment:
Cast is really unfortunate but I guess `DistanceSpacingMultiplier` was already a double before this branch, so I guess this'll have to live for the time being...

Reply:
Yeah, distances consistently follow the float data type, so distance spacing was probably supposed to be float, but unsure if it's simple changing that in the model class.

Reply:
I believe realm stores all number types internally using the same storage type. Should be a simple change, you're welcome to try it (may not even require a migration).